scheduler/scheduler_test.go

```diff
@@ -36,7 +45,9 @@ func TestInit(t *testing.T) {
 	if err := storeInstance.Init(); err != nil {
 		t.Fatal(err)
 	}
-	s := NewScheduler(storeInstance, &PluginFake{})
+	var ca security.CAAPI
```

Comment:
you can remove this variable and change the below `ca = &CAFake{}` to simply `ca := &CAFake{}` and all will be well. Do this everywhere. :)